docs/_layouts/default.html

```diff
@@ -8,8 +8,8 @@
       {% if page.bare %}
       {{ content }}
       {% else %}
+      {% include jlbp-nav.html %}
       <main role="main" id="page-content">
-        {% include jlbp-nav.html %}
         <section id="jump-content">
```

Comment:
I need to run the site again, but it looks like there's more navigation inside main, e..g jump-content? Maybe that's the rightand sidebar? Main is for the content only. This is important for accessibility and rendering on non-desktop screens.

Reply:
Done.